src/main/java/seedu/address/model/person/GenderType.java

```diff
@@ -0,0 +1,38 @@
+package seedu.address.model.person;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Enum class of a Person's gender in the address book.
+ * The 3 types of gender enum are: MALE, FEMALE, NA
+ */
+public enum GenderType {
+    MALE("Male"),
+    FEMALE("Female"),
+    NA("NA"); //NA is the default gender option
```

Comment:
Perhaps renaming `NA` to something like `NO_GENDER` would better indicate that this is just a placeholder for the gender field?